lib/elixir/test/elixir/kernel/binary_test.exs

```diff
@@ -264,13 +264,13 @@ defmodule Kernel.BinaryTest do
     assert <<1::size(foo.bar)>> = <<1::5>>
   end
 
-  defmacrop signed_16 do
+  defmacro signed_16 do
```

Comment:
Note: With `defmacrop`, I wasn't able to access the custom modifiers in `Code.eval_string` even by passing `__ENV__`.
Please let me know if there is a better way.